### PR TITLE
Update storage test suits for Windows

### PR DESCRIPTION
**Pull request by jingxu97**
Update storage test suits to enable some tests on Windows

Change-Id: I1f596bb9f7a0f41ae398aec43406ee5c753cdce3

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/storage/testsuites/multivolume.go [hunk 1]
```diff
@@ -30,12 +30,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
-	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 	e2evolume "k8s.io/kubernetes/test/e2e/framework/volume"
 	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
-	imageutils "k8s.io/kubernetes/test/utils/image"
+	//imageutils "k8s.io/kubernetes/test/utils/image"
```
Comment (Jiawei0227) [inline test/e2e/storage/testsuites/multivolume.go:37]: remove this?

File: test/e2e/storage/testsuites/util.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+// TODO: after issue https://github.com/kubernetes/kubernetes/issues/81245 is resolved
+// this utility can be moved to e2epod
+func getTestImage() int {
```
Comment (Jiawei0227) [inline test/e2e/storage/testsuites/util.go:28]: Sorry why is this related to https://github.com/kubernetes/kubernetes/issues/81245?
I think we can use utils from e2e/framework not the other way around?

**Comment by Jiawei0227** [comment]
Am I misunderstand something? I think we dont want too many utils everywhere. So the principle in my understanding is:
For storage related tests, it should go under `test/e2e/storage/utils`
For utils that are widely shared by other tests, it should go under `e2e/framework/volume` or `e2e/framework/pod` or `e2e/framework/util.go` depending on what the util function is.

Could you refactor it a little to remove the util or put it in the right place?

**Comment by jingxu97** [comment]
> Am I misunderstand something? I think we dont want too many utils everywhere. So the principle in my understanding is:
> For storage related tests, it should go under `test/e2e/storage/utils`
> For utils that are widely shared by other tests, it should go under `e2e/framework/volume` or `e2e/framework/pod` or `e2e/framework/util.go` depending on what the util function is.
> 
> Could you refactor it a little to remove the util or put it in the right place?

you are right, this function can be put under e2e/framework/volume. I don't remember what I did/test make me going this way. 
Also something wrong with this commit, I opened another PR https://github.com/kubernetes/kubernetes/pull/96479 and close this one.

**Pull request closed without merging**